docs/recipes/lighthouse-plugin-example/package.json

```
@@ -3,6 +3,6 @@
   "private": true,
   "main": "./plugin.js",
   "peerDependencies": {
-    "lighthouse": "^4.2.0"
+    "lighthouse": ">=5.2.0"
```

we might want to stick to
```suggestion
    "lighthouse": "^5.0.0"
```

as the recommended practice to avoid plugins being accidentally installed with incompatible changes later down the line with 6.x.x